homeassistant/components/motioneye/strings.json

```diff
@@ -9,6 +9,10 @@
           "surveillance_username": "Surveillance [%key:common::config_flow::data::username%]",
           "surveillance_password": "Surveillance [%key:common::config_flow::data::password%]"
         }
+      },
+      "hassio_confirm": {
+        "title": "motionEye via Home Assistant add-on",
+        "description": "Do you want to configure Home Assistant to connect to the motionEye service provided by the add-on: {addon}?"
```

Comment:
The unittest suggests "addon" may expand to "motionEye add-on", which makes this a little repetitive: "... provided by the add-on: motionEye add-on".

Reply:
I just put in "something". Will adjust it to the real-world example one I'm able to do end2end testing.

Reply:
Adjusted test strings to the real world.